### ISSUE TITLE
[BUG] SkiaSharp.Views.Forms Nuget file error - invalid check for GTK project

### ISSUE THREAD

**Issue by VladislavAntonyuk** [mono/SkiaSharp#1114]
The SkiaSharp.Views.Forms.WPF package needs to be installed in addition to the SkiaSharp.Views.Forms package in order to support Xamarin.Forms with WPF.	Project.GTK		

## Steps to reproduce
1. Create default net471 project
2. 
```
<Reference Include="System.Web" />
    <Reference Include="Mono.Posix" />
    <Reference Include="gtk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <Reference Include="gdk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <Reference Include="glib-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <Reference Include="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <Reference Include="pango-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <Reference Include="atk-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
    <PackageReference Include="SkiaSharp.Views.Forms.GTK" Version="1.68.2-preview.21" />
    <PackageReference Include="Xamarin.Forms.Platform.GTK" Version="4.6.0.191-nightly" />
```
3. Build application.

## Result
Build error

The issue is in SkiaSharp.Views.Forms.targets:
_RequiredWPFReferences is true.

**Comment by mattleibow** [comment]
I am not sure I understand this issue. I see that you say:
> The SkiaSharp.Views.Forms.WPF package needs to be installed in addition to the SkiaSharp.Views.Forms package in order to support Xamarin.Forms with WPF

Yes. To use WPF, you need the WPF packages?

Now, if this is a WPF project that has GTK, then you _do_ need to install the GTK packages because the check is possible too dumb and just checks to see if `gdk-sharp` is referenced.

What you can also do is just tell the compiler to not check anything and set this in the .csproj:

```xml
<ShouldCheckPlatformPackages>False</ShouldCheckPlatformPackages>
```

**Comment by VladislavAntonyuk** [comment]
I created a WinForms app, not WPF. I use new capris format, but set target platform to net471.
 I added all gtksharp reference (unfortunately I have to use full path to dll).
I have NetStandard library, which uses SkiaSharp Views Forms. 
Then I added package references to XF and SkiaSharp Vies Forms GTK to the main app 

As a result the app is not built 

I will try <ShouldCheckPlatformPackages>False</ShouldCheckPlatformPackages>
